test/python/primitives/test_estimator.py

```diff
@@ -127,7 +117,7 @@ def test_run_with_multiple_observables_and_none_parameters(self):
         circuit.cx(0, 1)
         circuit.cx(1, 2)
         with Estimator(circuit, ["ZZZ", "III"]) as est:
-            result = est(circuit_indices=[0, 0], observable_indices=[0, 1])
+            result = est(circuits=[0, 0], observables=[0, 1])
```

Comment:
Could you add a test that uses the deprecated named arguments, to make sure 
```python
est(circuit_indices=.., observable_indices=...)
```
still works correctly?